software/Predict.py

```diff
@@ -113,6 +113,8 @@ def get_variant_mapping(args, weights):
             #     mapping = KeyedDataSource.load_data(args.variant_mapping[0], "id", "rsid", value_white_list=set(weights.rsid))
         else:
             raise Exceptions.InvalidArguments("Unsupported variant mapping argument")
+    elif len(args.on_the_fly_mapping):
+        mapping = set(weights.rsid)
```

Comment:
@Heroico thanks for the clarification. 

The reason I set the mapping to a set is because I wanted to utilize this [function](https://github.com/hakyimlab/MetaXcan/blob/master/software/Predict.py#L122) but again when we are using `--on_the_fly_mapping` I am not sure I we can get a dict like object to pass to that function so I opted to use the set and leading to the changes below. 

I would appreciate suggestion to handle this better.